Sell/Product/Models/ProductModifierItem.swift

```diff
@@ -11,18 +11,21 @@ import Foundation
 import GraphQL
 #endif
 
-public struct ProductModifierItem: Codable, Equatable {
+public struct ProductModifierItem: Codable, Equatable, Hashable {
 
     public let id: String
+    public let status: ProductModifierItemStatus?
```

Comment:
Usually we decode to the raw String, and then have a computed property the converts the enum.
Reason being, if a new enum case is added by backend, the decoding will fail.
What do you think?

Reply:
I agree with you. And if I understand correctly, then the responsibility of converting string to enum (`ProductModifierItemStatus`) should be inside the frontier app. So when a new case is added, sdk can still return the raw String instead of `nil` for this `status` property.
I've updated `status` to String and removed the enum in sdk, please let me know if this is not the same as what you think. Thank you.